pkg/auth/authTokensGet.go

```diff
@@ -37,6 +38,48 @@ func GetTokens(
 	return err
 }
 
+func GetTokensByLoginId(
+	apiClient *galasaapi.APIClient,
+	console spi.Console,
+	loginId string,
+) error {
+
+	authTokens, err := getAuthTokensByLoginIdFromRestApi(apiClient, loginId)
+
+	if err == nil {
+		summaryFormatter := tokensformatter.NewTokenSummaryFormatter()
+
+		var outputText string
+		outputText, err = summaryFormatter.FormatTokens(authTokens)
+
+		if err == nil {
+			console.WriteString(outputText)
+		}
+	}
```

Comment:
This looks duplicated from the `GetTokens` function - would be good to reduce the code duplication